components/Tabs/Tabs.js

```diff
@@ -1,38 +1,43 @@
 class TabLink {
   constructor(element){
     // assign this.element to the element reference
-    this.element;
+    this.element = element;
+    this.info = this.element.dataset.info;
+
     // Get the tab data attribute and save the value here
-    this.tabData = ; 
+    this.tabData = `data-set=${this.info}` ; 
     // Find all elements with the .card class in index.html that correspond to the tab data attribute
+
+    this.cardElement = this.element.querySelector(`.card[data-set=${this.info}`)
     // If the data is 'all' then select all cards regardless of their data attribute
-    if(){
-      this.cards = ;
+    if('all'){
```

Comment:
Take a moment to think about this if statement. Right now it is only checking a string against nothing.